code/modules/urist/gamemodes/assault/equipment.dm

```diff
@@ -248,6 +248,7 @@
 	name = "box of frag mines (WARNING)"
 	desc = "<B>WARNING: These devices are extremely dangerous and can cause death within a short radius.</B>"
 	icon_state = "flashbang"
+	max_w_class = 3
```

Comment:
There's a proc to expand a box to contents size, it's probably better to call it in New() for these